chaoslib/litmus/cpu_hog/pod_cpu_hog.yml

```diff
@@ -92,14 +88,14 @@
       ## considering duration in milliseconds from spec
       ## with a grace period of 2 min before cpu stress job terminate
       set_fact:
-        job_wait_time: "{{ ((c_duration|int)/1000 + 120)| int }}"
+        job_wait_time: "{{ ((c_duration|int) + 120)| int }}"
```

Comment:
don't need to write int with 120 
it can be `"{{ ((c_duration | int) + 120 }}"`